packages/extension/src/newtab/ShortcutOptionsMenu.tsx

```diff
@@ -39,6 +39,10 @@ export default function ShortcutOptionsMenu({
     },
   ];
 
+  if (!React.isValidElement(ContextMenu)) {
+    return null;
+  }
```

Comment:
This is just a fix for SB purposes?

Reply:
Yes, unfortunately `dynamic` imports seems not working with the current sb setup and this is a fix for this (just don't load in Storybook)